data/data/openstack/variables-openstack.tf

```diff
@@ -306,6 +306,18 @@ variable "openstack_additional_network_ids" {
   default     = []
 }
 
+variable "openstack_additional_ports" {
+  type = list(list(object({
+    network_id = string
+    fixed_ips = list(object({
+      subnet_id  = string
+      ip_address = string
+    }))
+  })))
+  description = "Additional ports for each master node."
+  default     = [[], [], []]
+}
+
```

Comment:
Here, and below (`openstack_default_machines_port`, `openstack_machines_ports`) we define a Terraform data structure that mimics the input of a Terraform port. It allows multiple fixed_ips. It's the structure as defined in `tfvars/openstack/ports.go`